integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -693,11 +684,9 @@ public int rename(String oldPath, String newPath) {
     }
     try {
       mFileSystem.rename(oldUri, newUri);
-      synchronized (mOpenFiles) {
-        if (mOpenFiles.contains(PATH_INDEX, oldPath)) {
-          OpenFileEntry oe = mOpenFiles.getFirstByField(PATH_INDEX, oldPath);
-          oe.setPath(newPath);
-        }
+      if (mOpenFiles.contains(PATH_INDEX, oldPath)) {
```

Comment:
I think this behaves differently now, without the lock?

Reply:
yes, without lock, the entry for oldPath might be removed between these lines, added another check for whether `oe` is `null`.